doc/modules/core.rst

```diff
@@ -360,6 +360,9 @@ examples is seen below:
         ["random_spikes", "waveforms", "templates", "noise_levels"]
     )
 
+Note that any extension entered into the :code:`compute` function will be computed. Thus, even if an extension has already been computed
+it will be recomputed when :code:`compute` is called even if the parameters are the same.
```

Comment:
@zm711 I think it would be good here to mention how to check if an extension is computed and how to load it:

```suggestion
it will be recomputed when :code:`compute` is called even if the parameters are the same. 
To check if an extansion has already been computed and reload it, you can simply do:

.. code-block:: python

    if sorting_analyzer.has_extension("templates"):
        templates_extension = sorting_analyzer.get_extension("templates")
```

What do you think?

Reply:
Done :)

Reply:
I thought about that and then promptly forgot to mention it.